embassy-embedded-hal/src/shared_bus/asynch/spi.rs

```
@@ -97,11 +102,10 @@ where
 
         // On failure, it's important to still flush and deassert CS.
         let flush_res = bus.flush().await;
-        let cs_res = self.cs.set_high();
+        drop(cs_drop);
```

this will silently swallow the error.

I'd suggest doing `cs_drop.defuse()` and then do set_high as before so the error can be handled.

---

Changed as you advised. Could you please take a look again?